### ISSUE TITLE
Change border title on the fly

### ISSUE THREAD

**Issue by Conni2461** [mjlbach/neovim-ui#9]
I don't know if this is currently possible (atleast i don't see a way in plenary.border, etc) but it would be cool if we can change border title on the fly.

What i mean is this: 
- For telescope we are going to introduce a way where you can change you previewer on the fly to get a new view on the selected entry: https://github.com/nvim-telescope/telescope.nvim/pull/528
- Now i want to make clear which previewer currently displays. Would be cool if the title can show this.
- And because i can cycle between those previewers i want to update the title every time the previewer changes

I made add something for the time being in plenary or popup that makes this possible, it would be just cool if this will end up in the ui module as well so we can just easily update :)

Thanks for all your work :D

**Comment by mjlbach** [comment]
I don't think this should be hard, we have access to the border handle and can easily override the text

**Comment by Conni2461** [comment]
I also don't think it should be that hard. I just wanted to make sure it will end up here/vim.ui so we will have an easier time updating telescope :)

I think i will still add a version to plenary.nvim and popup.nvim because we need it right now.

**Comment by mjlbach** [comment]
Sounds good, I can just port over the changes or you can make a PR, the code is mostly identical still
